nucypher/blockchain/eth/actors.py

```diff
@@ -1674,21 +1665,20 @@ def assimilate(self, checksum_address: str, password: str = None) -> None:
 
         self.log.info(f"Resistance is futile - Assimilating Staker {original_form} -> {new_form}.")
```

Comment:
Perhaps it's a good moment to change this log message to something more in line with the current method. 
Something like this?
```suggestion
        self.log.info(f"Setting Staker from {original_form} to {new_form}.")
```

Reply:
✔️